From 904fed4b5a0b43386e47fde3b7f1f6a27d1a915e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20M=C3=A1gr?= <mmagr@redhat.com>
Date: Mon, 9 May 2016 22:37:30 +0200
Subject: [PATCH] Remove bundler dependency

---
 spec/helper.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spec/helper.rb b/spec/helper.rb
index 9b819db..d5818cc 100644
--- a/spec/helper.rb
+++ b/spec/helper.rb
@@ -1,6 +1,6 @@
 require 'spec_helper'
 require 'rubygems'
-require 'bundler/setup'
+#require 'bundler/setup'
 
 require 'em-http'
 require 'em-http/middleware/oauth2'
-- 
2.4.11

